### DISCUSSION TITLE
Advanced Skin Menu for Specific Mod Combinations

### DISCUSSION THREAD

**Discussion by takoz53** [ppy/osu#12977] [Ideas]
- Allows users to pre-set Skins for specific Mod combinations
- No more skin changing everytime you want to try other mods
- **Turn on/off hitlight** for each combination since this might be annoying in some skins
- **A search function** implemented to **find skins faster and easier**
- Lazy people can be even lazier!
- **Example:** 
Selected Combination: **HDDT**
Selected Skin for combination: **HDDTSkin2018**

I've come up with a template design of how everything should look like on the old client.
The template design should probably be able to explain what I've just been trying to explain here.
![finalproduct 1680x1050 ratio](https://user-images.githubusercontent.com/16865864/34509653-d6e58592-f04c-11e7-8e12-3e14147ca358.png)

**Comment by smoogipoo** [comment]
While we may consider having more skin customisation in the future, I'm backlogging + marking this as low prio since the rest of the game should come first.

**Comment by peppy** [comment]
Different skins for different mods isn't something I can get behind, personally. Making switching skins easy is as far as we should be going.

**Comment by Loreos7** [comment]
After 3 years I can say that it will be correct to make a shortcut that changes all of available skins one by one (now game has random skin select).

**Reply by frenzibyte** [reply]
I'm not sure I understand your comment? can you elaborate further and what that has to do with this discussion.

**Reply by Loreos7** [reply]
This has a lot to do with the discussion. It is suggested to add skin presets to the selected mods. I propose to solve this problem with a hotkey for alternating skin changes. Now game has a random skin change, it is not clear why.

**Reply by frenzibyte** [reply]
The game has a random skin change mostly for simple in-game feature testing on the skinning side of things. 

I don't believe adding hotkeys to switch to certain skins is the way to resolve this thread at all, but rather invent a whole other feature that don't resolve the points mentioned in the OP.

You should instead post a separate discussion thread for that.

**Reply by takoz53** [reply]
Alternating with a Hotkey does not really sound appealing and neccessary. It will only resolve the issues I mentioned partially (as you can still change skins), but is unwanted, as a hotkey or multiple hotkeys for multiple skins can just be frustrating to deal with.
